### PR TITLE
[PLT-6632] tests for Checkbox component

### PR DESCRIPTION
**Pull request by licarm**
## Description
tests for Checkbox Component

### CODE REVIEW

File: packages/riipen-ui/src/components/checkbox.test.jsx [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    });
+  });
+
+  describe("color prop", () => {
+    it("sets color class name with a valid color", () => {
+      const color = "secondary";
+
+      const wrapper = mount(<Checkbox color={color} readOnly />);
+
+      expect(toJson(wrapper)).toMatchSnapshot();
```
Comment (imogenem) [inline packages/riipen-ui/src/components/checkbox.test.jsx:73]: Remove snapshot test from color prop

File: packages/riipen-ui/src/components/checkbox.test.jsx [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+        <Checkbox required={required} label={label} readOnly />
+      );
+
+      expect(
+        wrapper
+          .find("Checkbox")
+          .find("label")
+          .text()
+          .substr(-1)
+      ).toEqual("*");
```
Comment (imogenem) [inline packages/riipen-ui/src/components/checkbox.test.jsx:157]: Nice!!

File: packages/riipen-ui/src/components/checkbox.test.jsx [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+      const warning = <span>warning!</span>;
+
+      const wrapper = mount(<Checkbox warning={warning} readOnly />);
+
+      expect(wrapper.childAt(0).contains(warning)).toEqual(true);
+    });
+  });
+
+  describe("focusVisible state", () => {
+    it("does not apply focusVisible class when component is rendered", () => {
```
Comment (imogenem) [inline packages/riipen-ui/src/components/checkbox.test.jsx:189]: Is this test necessary? I'm wondering if we only need to test after events occur.

**Review by imogenem — CHANGES_REQUESTED** [review]
Looks great! @jordanell can you comment on the focusVisible tests?

File: packages/riipen-ui/src/components/checkbox.test.jsx (hunk 3 above)
Comment (jordanell) [reply packages/riipen-ui/src/components/checkbox.test.jsx:189]: Ya I think we can safely remove this. Imo makes a good point.

**Review by jordanell — CHANGES_REQUESTED** [review]
Same comments as Imo :).

**Review by jordanell — APPROVED** [review]

**Pull request merged**